### ISSUE TITLE
[Logging v2] Make strip underscores optional in journald logging config

### ISSUE THREAD

**Issue by dbason** [rancher/rancher#33463]
Some logging endpoints (e.g. Elasticsearch) don't support fields with leading underscores.  The fluentbit config for journald logging should have the option to strip underscores.

**Comment by izaac** [comment]
We need to validate the config is properly set in the fluentbit config after chart install.

Edit YAML from Dashboard works, keeping the default `false` and also setting it as `true` 
`stripUnderscores: false`
`stripUnderscores: true`

Validate this is true for the `rke` source. Validate the Elasticsearch keeps working with `true` and `false`.
Then sanity check this value is set with the `rke2`, `k3s` sources.

**Comment by izaac** [comment]
Rancher version: `2.6 master-head (07/14/2021)` _58f6425_

RKE2 `v1.20.8+rke2r1` Imported using SELinux.
`stripUnderscores: false` did't add anything extra to the fluent-bit configuration and this default to `Off` according to the fluent-bit documentation. This is good.
`stripUnderscores: true`  adds the `Strip_Underscores On` line to the fluent-bit input configuration as expected. This is good too.

I checked the above on these scenarios:

- Installing with `stripUnderscores: false` 
- Installing with `stripUnderscores: true`
- Upgrading the installed chart with `stripUnderscores: false`
- Upgrading the installed chart with `stripUnderscores: true`



I did a sanity check on an imported k3s `v1.21.2+k3s1` cluster.

Configmap for k3s
`rancher-logging-k3s`

Configmap for rke2
`rancher-logging-rke2`
